lib/ui/views/sheets/dapp_sheet.dart

```diff
@@ -90,30 +99,31 @@ class DAppSheetState extends ConsumerState<DAppSheet>
     Future.delayed(Duration.zero, () async {
       var dappKey = widget.dappKey;
 
-      final networkSettings = ref.watch(
-        SettingsProviders.settings.select((settings) => settings.network),
-      );
-
-      final bridgeFlag = await ref.watch(
-        getFeatureFlagProvider(
-          networkSettings.network,
-          applicationCode,
-          featureCode,
-        ).future,
-      );
-      if (bridgeFlag == null || bridgeFlag == false || !DAppSheet.isAvailable) {
-        dappKey = '${dappKey}Ext';
+      if (widget.featureCode != null) {
```

Comment:
@redDwarf03 I don't  get it...

Reply:
The suffix `Ext` helps to find the url to launch (via launch button) if dapp is not managed directly in aeWallet.
See: https://testnet.archethic.net/explorer/transaction/000087F7890DB91FF666851B3D4BF618BBC4771218BA6A319AE3EF55AFA1718C2EA1